ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/components/build/clientlibs/.content.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
     jcr:primaryType="cq:ClientLibraryFolder"
-    categories="[backpack]"
+    categories="[backpackBuild]"
```

Comment:
For clientlib categories, it is more customary to use "domain-like" notation. I.e. if "backpack" is a generic category and "build" is sub, it goes as "backpack.build"

Reply:
Thanks for note about clientlib notation. Correction done.